obspy/signal/rotate.py

```diff
@@ -237,19 +226,22 @@ def rotate2ZNE(data_1, azimuth_1, dip_1, data_2, azimuth_2, dip_2, data_3,
     # Build transformation matrix.
     T = np.matrix([base_vector_1, base_vector_2, base_vector_3]).transpose()
 
-    # Apply it.
-    z, s, e = np.dot(T, [data_1, data_2, data_3])
-    # Replace all negative zeros. These might confuse some further processing
-    # programs.
+    if not inverse:
+        x, y, z = np.dot(T, [data_1, data_2, data_3])
+        y *= -1
+    else:
+        x, y, z = np.dot(np.linalg.inv(T), [data_1, -data_2, data_3])
```

Comment:
Not in general. I think the condition might be that the axes are orthogonal to each other.